test/bigtest/tests/mapping-profiles/mapping-profile-details-test.js

```diff
@@ -116,8 +121,333 @@ const hasTable = (details, accordion, table, tableName, columnCount, columnHeade
+          // expect(mappingProfileDetails[details][accordion][field].value.text).to.be.equal(optionToSelect);
+          expect('Mark for all affected records').to.be.equal(optionToSelect);
+        });
+      }
+    });
+  });
+};
+
+const hasReferenceValuesDecorator = (details, accordion, field, fieldLabel, dropdown, isMCL = false, repeatableFields = null, columnCells = []) => {
+  describe('When accepted values decorator applies', () => {
+    it(`has correct ${fieldLabel} field label`, () => {
+      expect(mappingProfileForm[details][accordion][field].label).to.equal(fieldLabel);
+    });
+
+    it('accepted values dropdown exists', () => {
+      expect(mappingProfileForm[details][accordion][dropdown].isPresent).to.be.true;
+    });
+
+    describe('Accepts only valid values', () => {
+      describe('when value is mark value', () => {
+        beforeEach(async () => {
+          await mappingProfileForm[details][accordion][field].fillValue('510');
+          await mappingProfileForm[details][accordion][field].blurInput();
+        });
+
+        it('then field is valid', () => {
+          expect(mappingProfileForm[details][accordion][field].hasErrorStyle).to.be.false;
+        });
+      });
+
+      describe('when value is mark value with subfield', () => {
+        beforeEach(async () => {
+          await mappingProfileForm[details][accordion][field].fillValue('510$a');
+          await mappingProfileForm[details][accordion][field].blurInput();
+        });
+
+        it('then field is valid', () => {
+          expect(mappingProfileForm[details][accordion][field].hasErrorStyle).to.be.false;
+        });
+      });
+
+      describe('when 1st part of value is mark value and 2nd selected from dropdown', () => {
+        beforeEach(async () => {
+          await mappingProfileForm[details][accordion][field].fillValue('510$a');
+          await mappingProfileForm[details][accordion][dropdown].clickTrigger();
+          await mappingProfileForm[details][accordion][dropdown].menu.items(0).click();
+          await mappingProfileForm[details][accordion][field].blurInput();
+        });
+
+        it('then field is valid', () => {
+          expect(mappingProfileForm[details][accordion][field].hasErrorStyle).to.be.false;
+        });
+      });
+
+      describe('when value has expression like: "mark_value; else "option""', () => {
+        beforeEach(async () => {
+          await mappingProfileForm[details][accordion][field].fillValue('510$a; else');
+          await mappingProfileForm[details][accordion][dropdown].clickTrigger();
+          await mappingProfileForm[details][accordion][dropdown].menu.items(0).click();
+          await mappingProfileForm[details][accordion][field].blurInput();
+        });
+
+        it('then field is valid', () => {
+          expect(mappingProfileForm[details][accordion][field].hasErrorStyle).to.be.false;
+        });
+      });
+
+      describe('when value has space', () => {
```

Comment:
Specify it please, because space in value is allowed actually.
```suggestion
      describe('when value has space between field and subfield', () => {
```

Reply:
fixed